near/src/config.rs

```diff
@@ -286,6 +285,89 @@ impl From<&str> for Config {
     }
 }
 
+#[easy_ext::ext(GenesisConfigExt)]
```

Comment:
what is this for?

Reply:
I extracted GenesisConfig into a separate crate, but some of the methods (related to testing and only `near` binary) were hard and pointless to migrate, so I split them into basic (moved them into the crate) and "extension" (left here, but to do so I need a trait and instead of writing that duplicative trait signature, I used this tiny `easy_ext` crate, which essentially just creates the trait by a given impl)